config/ext/rspec.yml

```diff
@@ -435,32 +445,23 @@ RSpec/ExpectInHook:
   VersionAdded: '1.16'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ExpectInHook
 
+RSpec/ExpectInLet:
+  Description: Do not use `expect` in let.
+  Enabled: true
+  VersionAdded: '2.30'
+  Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ExpectInLet
+
 RSpec/ExpectOutput:
   Description: Checks for opportunities to use `expect { ... }.to output`.
   Enabled: true
   VersionAdded: '1.10'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/ExpectOutput
 
-RSpec/FilePath:
```

Comment:
Wurde entfernt in 3.0
https://github.com/rubocop/rubocop-rspec/releases/tag/v3.0.0